radis/test/lbl/test_factory.py

```diff
@@ -746,7 +744,7 @@ def test_vaex_and_pandas_spectrum_noneq():
         "mole_fraction": 0.1,
         "wstep": "auto",
         "path_length": 1,
-        "databank": "hitemp",
+        "databank": "hitran",
```

Comment:
we actually already download HITEMP CO for other tests, so you can keep this one !